packages/web-client/app/utils/web3-strategies/layer1-chain.ts

```diff
@@ -34,7 +34,9 @@ import {
 import { task } from 'ember-concurrency';
 import { taskFor } from 'ember-concurrency-ts';
 import { action } from '@ember/object';
+import { toWei } from 'web3-utils';
 
+const APPROXIMATE_WITHDRAWAL_GAS_AMOUNT = new BN('290000');
```

Comment:
eventually this will be a token address specific amount. should the SDK be responsible for maintaining this value?

Reply:
Interesting. I agree -- I will make this change. However, that surfaces a problem. At this point in the withdrawal workflow, the user has not identified which token they want to withdraw. Do you think we need to take this back to the design team?

Reply:
Perhaps, it's probably related to this as well https://linear.app/cardstack/issue/CS-1597/figure-out-how-to-support-multi-safe-txn-call-for-multiple-token